ChangeLog

```diff
@@ -74,7 +74,7 @@
  * #1805 (Brent's implementation has a stability issue)
  * #1807 (Beta::computeCharacteristicFunction is wrong if the lower bound is not zero)
  * #1815 (1.16 fails with dlib-cpp-19.22)
-
+ * #1820 (Incoherent results in LinearModelAnalysis)
```

Comment:
can you mention the issue in the commit so it will be closed ?

Reply:
The issue is already linked to this PR